src/bundles/ipfs-provider.js

```diff
@@ -315,6 +315,25 @@ const selectors = {
  */
 
 const actions = {
+
+  doSetupLocalStorage: () => async () => {
+    /** For the Explore page (i.e. ipld-explorer-components) */
+    const useRemoteGatewaysToExplore = localStorage.getItem('explore.ipld.gatewayEnabled')
+    if (useRemoteGatewaysToExplore === null) {
+      // by default, disable remote gateways for the Explore page (i.e. ipld-explorer-components)
+      await writeSetting('explore.ipld.gatewayEnabled', false)
+    }
+
+    const kuboGateway = readSetting('kuboGateway')
+    if (kuboGateway === null || typeof kuboGateway === 'string' || typeof kuboGateway === 'boolean' || typeof kuboGateway === 'number') {
+      // empty or invalid, set defaults
+      await writeSetting('kuboGateway', { trustlessBlockBrokerConfig: { init: { allowLocal: true, allowInsecure: true } } })
```

Comment:
nit: shouldn't we store port and host? user may have it on something other than `127.0.0.1:8080`, for example ex-brave user could have `127.0.0.1:48080`

Reply:
That happens elsewhere so i was planning to leave that alone

Reply:
I guess that would work for now but it doesn't feel great